### ISSUE TITLE
[V3] Implement new Concept Exercise: equality

### ISSUE THREAD

**Issue by kristinaborn** [exercism/javascript#954]
This issue describes how to implement the `equality` concept exercise for the JavaScript track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This exercise aims to teach students how to compare values for equality.

## Learning objectives

- Know how to use the abstract equality operator ("double equals")
- Know how to use the strict equality operator ("triple equals")
- Know how to use `Object.is()`
- Understand the difference in how primitives are handled by each of the value-comparison operations

## Out of scope

- `SameValueZero`
- Deep object comparison

## Concepts

- `equality`

## Prerequisites

- `strings`
- `numbers`
- `booleans`
- `objects`
- `type-conversion`

### After

- `SameValueZero`
- Which equality algorithm is used by common built-in methods like `Array.indexOf` and `Array.includes`
- When to use `Object.is()` over strict equality comparison

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by shubhsk88** [comment]
@SleeplessByte  How is it going with this ? How much work got done in this? Sorry was busy with personal life. I would like to talk this back on

**Comment by junedev** [comment]
We decided to combine the concept "comparison" (which includes equality) and "conditionals" into one exercise and that exercise was already done (https://github.com/exercism/javascript/pull/1037). So I am closing this issue.

@shubhsk88 Sorry nobody got back to you earlier. If you still want to help out with V3, have a look at the more recent open issues.
There are some nice smaller ones like https://github.com/exercism/javascript/issues/1019 or if you can manage something bigger maybe https://github.com/exercism/javascript/issues/1049 would suit you.
